dashboard/index.js

```diff
@@ -26,16 +25,43 @@ const DEFAULT_SCROLL_OPTIONS = {
 };
 
 class Dashboard {
-  constructor(options) {
+  constructor(options) { // eslint-disable-line max-statements
+    // Options, params
     options = options || {};
     const title = options.title || "webpack-dashboard";
 
     this.color = options.color || "green";
     this.minimal = options.minimal || false;
-    this.setData = this.setData.bind(this);
-
     this.stats = null;
 
+    // Data binding, lookup tables.
+    this.setData = this.setData.bind(this);
+    this.actionForMessageType = {
```

Comment:
I moved this up to save us from continually re-creating and setting this object below.